integration/test_magic_folder.py

```diff
@@ -383,10 +384,11 @@ def test_edmond_uploads_then_restarts(reactor, request, temp_dir, introducer_fur
     # re-starting edmond right now would "normally" trigger the 2880 bug
 
     # kill edmond
-    edmond.signalProcess('TERM')
-    yield edmond._protocol.exited
+    edmond.transport.signalProcess('TERM')
+    yield edmond.transport.exited
     time.sleep(1)
```

Comment:
Arbitrary sleep ? Dont know the full logic yet but wondering if this could be removed.